content/blog/2024/2024-06-06-junit5-functional-interfaces.md

```diff
@@ -0,0 +1,620 @@
+---
+authors: [sagaofsilence]
+categories: [Java]
+date: 2024-06-06 00:00:00 +1100
+excerpt: Guide to JUnit 5 Functional Interfaces.
+image: images/stock/0041-adapter-1200x628-branded.jpg
+title: Guide to JUnit 5 Functional Interfaces
+url: junit5-functional-interfaces
+---
+
+In this article, we would get familiar with JUnit 5 functional interfaces. JUnit 5 significantly advanced from its predecessors. Features like functional interfaces can greatly simplify our work once we grasp their functionality.
```

Comment:
Let's add the link to the GitHub repository. I'll update the link once the PR gets merged.

Reply:
added link